query/sources/common/src/main/java/org/codice/ddf/admin/sources/utils/RequestUtils.java

```diff
@@ -72,21 +72,20 @@
      * @param urlField  the original request url
      * @return {@link Response} of the request
      */
-    public ReportWithResultImpl<ResponseField> sendGetRequest(WebClient webClient,
+    public Report<ResponseField> sendGetRequest(WebClient webClient,
             UrlField urlField) {
-        ReportWithResultImpl<ResponseField> responseResult = new ReportWithResultImpl<>();
-        responseResult.addMessages(endpointIsReachable(urlField));
-        if (responseResult.containsErrorMsgs()) {
+        Report<ResponseField> responseResult = Reports.from(endpointIsReachable(urlField));
+        if (responseResult.containsErrorMessages()) {
             return responseResult;
         }
 
         try {
             Response response = webClient.get();
-            responseResult.result(responseFieldFromResponse(response, urlField));
+            responseResult.setResult(responseFieldFromResponse(response, urlField));
```

Comment:
Line 84-85 should be able to be `return Reports.from(responseFieldFromResponse(response, urlField));`